src/main/java/org/example/repository/UserRepository.java

```diff
@@ -0,0 +1,10 @@
+package org.example.repository;
+
+import org.example.model.User;
+
+import java.util.Optional;
+
+public interface UserRepository {
+    void save(User user);
+    Optional<User> getById(Long id);
```

Comment:
In SpringBoot Data JPA, we have the following namings:
```suggestion
    Optional<User> findById(Long id);
    User getById(Long id);
```